common/JackTools.cpp

```diff
@@ -27,6 +27,7 @@
 #include <signal.h>
 
 #ifdef WIN32
+#include <JackPlatformPlug_os.h>
```

Comment:
If "WIN32" is not defined on visual c, why have you added this line?

Reply:
This was from before I learned about the stability of _WIN32, I should've fixed this. (WIN32 works randomly still, but very unrelyable).